extensions/wikia/Recirculation/js/helpers/sponsoredContent.js

```diff
@@ -38,9 +39,28 @@ define('ext.wikia.recirculation.helpers.sponsoredContent', [
 	}
 
 	function getApplicableContent(sponsoredContent) {
-		return sponsoredContent.filter(function (el) {
-			return !el.geos.length || el.geos.indexOf(userGeo) !== -1;
+		var geoSpecificContent = sponsoredContent.filter(function (el) {
+			return (!el.geos.length || el.geos.indexOf(userGeo) !== -1) && (!el.wikiIds.length || el.wikiIds.indexOf(w.wgCityId) !== -1);
 		});
+		var siteSpecificContent = sponsoredContent.filter(function (item) {
```

Comment:
here we have `item` but in first func `el` is defined. this can be unified.

also - shouldn't we have `item.wikiIds.length` check here as well?